_markbind/navigation/site-nav.md

```diff
@@ -1,3 +1,3 @@
 <navigation>
-* [User Guide]({{ baseUrl }}/index.html)
+* [Home]({{ baseUrl }}/index.html)

```

Comment:
What are we planning to put in `Home`? Cuz now like when it starts up its just blank and says home. Should we just have the `ug` as the default?

Reply:
I think we can fill in like a brief description of our product like what [Reposense](https://reposense.org/) has done XD

Reply:
Ahh okay cool!